packages/core/src/types.ts

```diff
@@ -1911,6 +1911,12 @@ export interface Subscription {
   unsubscribe(): void;
 }
 
+export type Selection<TSelected> = Readable<TSelected>;
```

Comment:
this is unused, do we need this alias? it probably could be dropped

Reply:
Yes, good catch! Removing it.